clusters/production/overlay/third-party/ingress-nginx/ingress-nginx.yaml

```diff
@@ -48,6 +48,31 @@ spec:
               service:
                 annotations:
                   service.beta.kubernetes.io/azure-load-balancer-resource-group: common
+              config:
+                custom-http-errors: "503"
+            defaultBackend:
+              enabled: true
+              resources:
+                limits:
+                  cpu: "3"
+                  memory: 1500Mi
+                requests:
+                  cpu: 300m
+                  memory: 1000Mi
```

Comment:
```suggestion
                limits:
                  cpu: 50m
                  memory: 100Mi
                requests:
                  cpu: 50m
                  memory: 50Mi
```

Reply:
Reduce mem+cpu request and limit.
Should probably try this on dev clusters first